Rakefile

```diff
@@ -46,9 +46,11 @@ end
 desc 'Grab libs'
 task 'setup_libs' do
   in_venv = system "python -c \"import sys ; exit(not hasattr(sys, 'real_prefix'))\""
-  raise 'Not in dev venv/CI environment - bailing out.' if !in_venv && !ENV['CI']
+  raise 'Not in dev venv/CI environment/Integrations - bailing out.' if !in_venv && !ENV['CI'] && !ENV['SDK_HOME']
```

Comment:
So this would break `setup_libs` for the dd-agent. I think we should change this - correct me if I'm wrong.